<launch>
  <!-- rslidar raw packets node -->
  <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
    <!--param name="pcap" value="/home/tony-sti/lidar_data/lm75-170616-roadtest.pcap"/-->
    <!--param name="device_ip" value="192.168.1.200"/-->
    <!-- Set the Lidar data port on the PC side, default is 6699 -->
    <param name="port" value="6699"/>
  </node>
  
  <!--  rslidar pointcloud parse node -->
  <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
    <param name="curves_path" value="/data/LM128/curves.csv"/>
    <param name="angle_path" value="/data/LM128/angle.csv"/>
    <param name="channel_path" value="/data/LM128/ChannelNum.csv"/>
  </node>
  
  <!-- gps data record node -->
  <node name="gps" pkg="sensors" type="findm"   >
    <param name="port" value="/dev/ttyUSB1" />
    <param name="baud" value="115200" />
    <param name="appkey" value="510328" />
    <param name="appSecret" value="a9a55c74fa8797e5ba9ca482d83311b9a6341ab99996050345ad43a480bfded9" />
    <param name="deviceId" value="123343455443" />
    <param name="deviceType" value="gps" />

  </node>
  
  <!-- nav440 imu data record node -->
  <node  name="nav440" pkg="sensors" type="nav440"  >
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud" value="38400" />
  </node>
   


  <!-- wheel encoder data record -->
  <node  name="encoder" pkg="sensors" type="encoder"  >
    <param name="port" value="/dev/ttyUSB2" />
    <param name="baud" value="115200" />
  </node>
  
  <!-- rosbag record sensor data -->
  <!--node name="record" pkg="rosbag" type="record" output="screen" args="-O /home/sti/20170704.bag /nav440/nav440 /gps/gps /canbus/canbus /rslidar_node/rslidar_packets" /-->
  
</launch>
